src/app.html

```diff
@@ -10,6 +10,9 @@
 		<link rel="preconnect" href="https://fonts.googleapis.com">
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 		<link href="https://fonts.googleapis.com/css2?family=Fira+Sans+Extra+Condensed:wght@800&family=Fira+Sans:wght@300&display=swap" rel="stylesheet">
+
+		<!-- TODO: load this via package -->
+		<script src="https://cdn.jsdelivr.net/npm/@alch/alchemy-web3@latest/dist/alchemyWeb3.min.js"></script>
```

Comment:
What issue we running into loading this via normal routes? This the node specific function issue?

Reply:
Yeah I messed with it for a bit but didn't get installing via package to work yet